operator/deploy/services.yaml

```diff
@@ -56,7 +56,7 @@ spec:
   selector:
     role: webconsole
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
To update the version to apps/v1, operator requires some changes. Operator uses client-go-1.5 api and written for Kubernetes 1.8 and apps/v1beta2 is the latest available in 1.8. 
I quickly tried migrating operator to 1.9 but there seems to be a major change and that is breaking the operator. I will figure out what's breaking and  migrate the operator to kubernetes 1.9 and then we can merge this change.